### ISSUE TITLE
Unable to render Bool value

### ISSUE THREAD

**Issue by amitsharma2705** [GriddleGriddle/Griddle#335]
I don't its a bug from griddle side or react but I am unable to render bool value (false) got from webapi.

Change : coldata = coldata.toString() in gridRow.jsx.js render function 
after this seems to working for me

**Comment by YacheLee** [comment]
I have the same problem, too.

**Comment by gregsantos** [comment]
@amitsharma2705 coldata?

**Comment by ryanlanciaux** [comment]
All -- I realize this issue is from some time ago but we recently have released Griddle 1.0 and are trying to make sure we are a bit better about responding to issues.

The following is for Griddle 1.0:

I would recommend creating a component as follows: 

```
const boolData = ({value}) => <span>{ value.toString() }</span>;
```
and passing this in as a custom component on the column that contains boolean data.

```
<Griddle data={localData} plugins={[LocalPlugin]}>
  <RowDefinition>
    <ColumnDefinition id="name" customComponent={boolData} />
    <ColumnDefinition id="location.state" nested={true}/>
  </RowDefinition>
</Griddle>
```
Closing this one for now but please feel free to create a new issue if any problems are encountered. 👍
